pages/api/configuration.ts

```diff
@@ -14,15 +14,28 @@ import {
 } from "../../generated/graphql";
 
 const CONFIGURATION_KEYS = [
-  "THAT_SECRET",
-  "THIS_VALUE",
+  "PUBLIC_TOKEN",
+  "CUSTOMER_CREATED_METRIC",
+  "FULFILLMENT_CREATED_METRIC",
+  "ORDER_CREATED_METRIC",
+  "ORDER_FULLY_PAID_METRIC",
 ];
 
-const filterConfigFromMetadata = (input: MetadataInput[] | MetadataItem[]) =>
+const prepareMetadataFromRequest = (input: MetadataInput[] | MetadataItem[]) =>
```

Comment:
I'm thinking about moving these functions to `lib/metadatada.ts` in template (but wil do that in separate pull request).

Reply:
Once it's done, I'll extract from this to the `saleor-app-sdk`